scm/driver/bitbucket/git.go

```diff
@@ -207,14 +207,12 @@ func convertCommit(from *commit) *scm.Commit {
 			Name:   from.Author.User.DisplayName,
 			Email:  extractEmail(from.Author.Raw),
 			Date:   from.Date,
-			Login:  from.Author.User.Username,
```

Comment:
What happened to these fields?

Reply:
Bitbucket changed their api and removed the "login" field:
https://developer.atlassian.com/cloud/bitbucket/bitbucket-api-changes-gdpr/#removal-of-usernames-from-user-referencing-apis